### PR TITLE
Issue #3059646: Consider upgrading field_group module

### PR DESCRIPTION
**Pull request by ol0lll**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by ol0lll** [comment]
The field_group module provides an update hook, setting the 'region' key for each field_group.

**Comment by ol0lll** [comment]
* Missing state handling: https://www.drupal.org/project/field_group/issues/3013395
* Field group in paragraphs: https://www.drupal.org/project/paragraphs/issues/2907094

**Review by dbosen — CHANGES_REQUESTED** [review]
The inline form display of image media entity is broken now. Can be seen, when you open an image entity paragraph for editing, all the input fields are differently layouted than before.
Could also be the case on other form displays, but I did not find any.

**Comment by ol0lll** [comment]
Patch from https://www.drupal.org/i/2907094 fixes field_groups which are directly in paragraphs, a use case we don't implement.
Patch from https://www.drupal.org/i/2986704 fixes field_groups in IEF - applied that.

File: composer.json [hunk 1]
```diff
@@ -109,7 +113,7 @@
         "drupal/entity_browser": "1.7",
         "drupal/entity_reference_revisions": "^1.0",
         "drupal/fb_instant_articles": "^1.0",
-        "drupal/field_group": "^1.0",
+        "drupal/field_group": "^3.0",
```
Comment (chrfritsch) [inline composer.json]: We have to fix the version.

File: drupal-org.make [hunk 2]
```diff
@@ -48,7 +48,7 @@ projects[entity_reference_revisions][version] = 1
 projects[fb_instant_articles][type] = module
 projects[fb_instant_articles][version] = 1
 projects[field_group][type] = module
-projects[field_group][version] = 1
+projects[field_group][version] = 3
```
Comment (chrfritsch) [inline drupal-org.make]: We have to fix the version.

Comment (chrfritsch) [reply drupal-org.make]: Field group and paragraphs patch are missing

**Review by chrfritsch — CHANGES_REQUESTED** [review]

**Review by chrfritsch — DISMISSED** [review]
Looks good to me

**Comment by chrfritsch** [comment]
Let's wait for the new paragraphs release now. Then we can get rid of the additional paragraphs patch

**Pull request merged**